 <launch>
<!--Input topic arguments-->
<arg name="marker_topic_name" default="/marker_transform"/> 

<!--Output topics arguments-->
<arg name="stiffness_topic_name" default="/stiffness_command"/> 

<!--Tf arguments-->  
<arg name="base_frame"          default="base_footprint"/>
<arg name="end_effector_frame"          default="wrist_ft_tool_link"/>

<!--Tune parameters-->
<arg name="min_robot_stiffness" default="100"/>
<arg name="max_robot_stiffness" default="500"/>
<arg name="lambda_min" default="0.05"/>
<arg name="lambda_max" default="0.25"/>
<arg name="window_length" default="50"/>



  <node pkg="stiffness_learning" name="stiffness_learning" type="stiffness_learning" output="screen">
    <!--Input topic-->
    <param name="marker_topic_name" value="$(arg marker_topic_name)"/>

    <!--Output topics-->
    <param name="stiffness_topic_name" value="$(arg stiffness_topic_name)"/>

    <!--Tf arguments--> 
    <param name="base_frame_name"     value="$(arg base_frame)"/>
    <param name="ee_frame_name"     value="$(arg end_effector_frame)"/>

    <!--Tune parameters-->
    <param name="stiffness_min"     value="$(arg min_robot_stiffness)"/>
    <param name="stiffness_max"     value="$(arg max_robot_stiffness)"/>
    <param name="lambda_min"        value="$(arg lambda_min)"/>
    <param name="lambda_max"        value="$(arg lambda_max)"/>
    <param name="window_length"     value="$(arg window_length)"/>
  </node>
 </launch>